cinema_service/settings.py

```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = ("django-insecure-6vubhk2$++agnctay_4pxy_8cq)"
+              "mosmn(*-#2b^v4cgsh-^!i3")
```

Comment:
The `SECRET_KEY` should be kept secret in production environments. Consider using environment variables or a secrets manager to handle this securely.